core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -139,6 +147,40 @@ class Translations : LinkedHashMap<String, TranslationEntry>(){
         return percentComplete
     }
 
+    private fun calculatePercentageForNationsFile(language: String, notTranslatedNations: Array<Nation>): Pair<Int, Int> {
```

Comment:
Please call the second variable originalNations or something, notTranslated has a different meaning in this context =)

Reply:
Done.